faculty/clients/experiment.py

```diff
@@ -188,7 +202,12 @@ def list(self, project_id):
         return self._get(endpoint, ExperimentSchema(many=True))
 
     def create_run(
-        self, project_id, experiment_id, started_at, artifact_location=None
+        self,
+        project_id,
+        experiment_id,
+        started_at,
+        artifact_location=None,
+        tags=[],
```

Comment:
Having mutable objects as default arguments is discouraged in Python for reasons I'd rather not type out (can explain in person). Recommend having `tags=None` here and then in the method have:

```python
if tags is None:
    tags = []
```